### DISCUSSION TITLE
Sync specific folder outside of "OneDrive" Rootfolder to specific OneDrive folder?

### DISCUSSION THREAD

**Discussion by bjspi** [abraunegg/onedrive#2687] [Q&A] [answered]
Hi all,

I hope this is not yet answered - at least I couldnt find it described in the Docs. I understand that the OneDrive usually uses one main folder as root, which corresponds with the root of your OneDrive Cloud root. If I want to upload / download only a particular folder inside this, I can obviously use the sync_list file - it works for me.

But what if I want to sync a particular folder which is not part at all of the local OneDrive directory, to a particular subdirectory in my OneDrive root? Is that possible?

**My Usecase is**: I have a few tools and scripts which maintain their own folder structure directly within the specific tools-directory, which is not changeable. So for Tool 1 it might be: /workspace/tool1/subdir/subdir2, for tool2 vice versa. And I only want the subdir2 of tool1 and tool2 to by synced to a specific OneDrive target folder (eg OneDriveCloudRoot/Tool1_tempfiles).

Right now I'm creating symlinks for the requierd files into my OneDrive local Directory, but that is rather ugly I suppose. 

Thank you in advance!

**Comment by abraunegg** [comment, marked as answer]
@bjspi 
This question is constantly being asked, most recently Feb28, 2024: https://github.com/abraunegg/onedrive/discussions/2649

Within your 'sync_dir' local tree (usually `~/OneDrive`) - symbolically link the folders in the location you want, so that they are synced online into that location.

**Reply by bjspi** [reply]
Maybe it's worth mentioning this under advanced usage in the Docs.
"Sync folders outside of OneDrive Root" to OneDrive. :D 

Thank you very much!
